docs/cluster/networking.md

```diff
@@ -0,0 +1,142 @@
+# Networking in Kubernetes
+
+## Docker model
+
+Before discussing the Kubernetes approach to networking, it is worthwhile to
+review the "normal" way that networking works with Docker.  By default, Docker
+uses host-private networking.  It creates a virtual bridge, called `docker0` by
+default, and allocates an [RFC1918](https://tools.ietf.org/html/rfc1918) range
+for that bridge.  Each container that Docker creates is allocated a virtual
```

Comment:
IIUC, the veth is added to the bridge, like `brctl addif br0 veth1`.  So, maybe reverse how this is phrased. e.g.:

Each time docker creates a container, it allocates a virtual ethernet device (called `veth`) on the bridge.   The `veth` is mapped to appear as `eth0` in the container, using Linux namespaces.